org-roam-macs.el

```diff
@@ -55,7 +55,9 @@ If FILE, set `org-roam-temp-file-name' to file and insert its contents."
     `(let ((,current-org-roam-directory org-roam-directory))
        (with-temp-buffer
          (let ((org-roam-directory ,current-org-roam-directory)
-               (org-mode-hook nil))
+               (org-mode-hook nil)
+               (org-startup-with-latex-preview nil)
```

Comment:
simply setting `(org-inhibit-startup t)` should save more time:

https://github.com/emacs-straight/org-mode/blob/a1e5bee5cb9c34ceb8226597605a49638bee7cec/lisp/org.el#L4883

Reply:
https://github.com/org-roam/org-roam/pull/1057/commits/b2de9f4b76423010aa1c8c2a00bb7d05e7fac9b3